src/diffusers/loaders/unet.py

```diff
@@ -686,6 +686,13 @@ def _load_ip_adapter_weights(self, state_dict):
         # because `Resampler` also has `attn_processors`.
         self.encoder_hid_proj = None
 
+        # store original attn_processors if not already stored
+        if (
+            getattr(self, "original_attn_processors", None) is None
+            or getattr(self, "original_attn_processors", None) == {}
+        ):
+            self.original_attn_processors = self.attn_processors
```

Comment:
```suggestion
```

Can we maybe not add this mechanism for now? When unloading the IP adapters let's just set the default SDPA (torch 2.0) attention processors. This introduces too much magic for now IMO